### ISSUE TITLE
User not able to see my messages but I can see his

### ISSUE THREAD

**Issue by jared2122** [vector-im/element-web#9756]
### Description

I've realized for the last week or two that my conversations with one Matrix user (@jeffnewbold:matrix.org) have been one-sided: I can see his messages but he cannot see mine. I also tested this in a room with Jeff and @matthew:matrix.org. In that room, I could read everything, but Jeff still couldn't see my messages even though Matthew could. 

These are the room URLs: 
https://riot.im/app/#/room/!VJTjzbFiYgsDonDmll:matrix.org
https://riot.im/app/#/room/!RidtQlciNyaBoFwOMg:matrix.org

### Steps to reproduce

Log was sent  

![Ghost](https://user-images.githubusercontent.com/47565487/57959862-764d9180-78cb-11e9-96d9-87623a3e4f85.PNG)

### Version information

Both web (1.1.2) and iOS.
Chrome browser
Windows OS
https://riot.im/app

**Comment by jared2122** [comment]
The other user associated with this issue cleared his cache and that appeared to fix the problem.

**Comment by jryans** [comment]
I am glad to hear that clear cache resolved the issue. We'll track this for future investigation so that ideally such workarounds won't be needed.

**Comment by yousefamar** [comment]
This is probably a different cause to what happened above, but in case someone gets here via Google, I ran into it too: I have a room on my homeserver (matrix.amar.io, #room@amar.io) with people from my homeserver and people from matrix.org. At some point my server ran out of disk space (probably contributed to by #matrix:matrix.org) and I shut it down briefly to upgrade the storage. When I started it back up, I think anything federation-related didn't work, WebRTC didn't work (self-hosted coturn), and requesting keys to decrypt messages from people outside amar.io didn't work.

In that room, the people on amar.io could see everything, the people on matrix.org could see messages from amar.io, people from matrix.org could see the messages from others on matrix.org, but the people on amar.io could not see messages from matrix.org. From the perspective of amar.io users, the matrix.org users just went offline for good. From the perspective of matrix.org users, the amar.io folks were ignoring them.

Finally, I tracked down the issue to unsaved firewall settings (`ufw`) so the ports for federation and for TURN were blocked. I think that perhaps to avoid this, there could be some checker built into element that acts like the [Federation Tester](https://federationtester.matrix.org/) and warns users that their message may not have been seen? Ultimately it's down to the server admins to keep things running right, but for the sake of good UX.
